mempool/v1/mempool.go

```diff
@@ -646,7 +661,7 @@ func (txmp *TxMempool) handleRecheckResult(tx types.Tx, checkTxRes *abci.Respons
 		"code", checkTxRes.Code,
 	)
 	txmp.removeTxByElement(elt)
-	txmp.metrics.FailedTxs.Add(1)
+	txmp.metrics.FailedTxs.With(mempool.FailedRecheck).Add(1)
```

Comment:
Maybe want to add the tracer to this line as well so we can monitor rechecks

Reply:
cool addressed in https://github.com/celestiaorg/celestia-core/pull/1158/commits/6cf6d598b204603a956031b82023afae32126cf7